docs/man/ctags-faq.7.rst

```diff
@@ -0,0 +1,391 @@
+.. _ctags-faq(7):
+
+==============================================================
+ctags-faq
+==============================================================
+--------------------------------------------------------------
+Universal Ctags FAQ
+--------------------------------------------------------------
+:Version: 5.9.0
+:Manual group: Universal Ctags
+:Manual section: 7
+
+This is the Universal Ctags FAQ (Frequently-Asked Questions).
+It is based on `Exuberant Ctags FAQ <http://ctags.sourceforge.net/faq.html>`_
+
+.. contents::
+
+DESCRIPTION
+-----------
+
+.. TODO: https://github.com/universal-ctags/ctags/issues/2312
+	#1421: feature: clean up stale tags when appending (`-a`)
+	#2356: can't pre-process the macro but it works with Exuberant Ctags 5.8
+	#2540: C/C++：conditional compilation like #ifdef will cause parse errror
+
```

Comment:
I would like you to add an item about ~/.ctags at the top.
e.g.

```
What is the difference between Universal Ctags and Exuberant Ctags?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Universal Ctags is an unofficial fork of Exuberant Ctags.
The differences are summarized in ctags-incompatibilities(7) man page.
The most notable one is Universal Ctags doesn't read `.ctags` file under your home directory.
Instead, it reads `*.ctags` under `~/.ctags.d` directory.
```

Reply:
Referencing to `ctags-incompatibilities(7)` is a very good idea.
fixed.